core/backend/src/IModelDb.ts

```diff
@@ -890,6 +890,59 @@ export abstract class IModelDb extends IModel {
     return nativeDb;
   }
 
+  /**
+   * Determines if the schemas in the Db must or can be upgraded by comparing them with those included in the
+   * current version of the software.
+   * @param pathName Path of the DgnDb file

```

Comment:
say "file name of the briefcase"

Reply:
Fixed now - see new comment and argument name.